devices/Pcd8544/README.md

```diff
@@ -0,0 +1,178 @@
+            var backlightPin = 21;
+
+            var gpioController = new GpioController();
+            var spiConnectionSettings = new SpiConnectionSettings(1, 5)
+            {
+                ClockFrequency = 5_000_000,
+                Mode = SpiMode.Mode0,
+                DataFlow = DataFlow.MsbFirst,
+                ChipSelectLineActiveState = PinValue.Low
+            };
+            var spiDevice = new SpiDevice(spiConnectionSettings);
+            var pwmChannel = PwmChannel.CreateFromPin(backlightPin);
+
+            var lcd = new Iot.Device.Pcd8544(dataCommandPin, spiDevice, resetPin, pwmChannel, gpioController, false);
+```
+
+If you don't want neither a PWM neither a reset pin, you can then pass a negative pin number for reset and null for the PWM:
+
+```csharp
+lcd = new(27, spi, -1, null);
+
+var lcd = new Iot.Device.Pcd8544(dataCommandPin, spiDevice, -1, null);
+```
+
+Note that there is as well the possibility to pass a normal pin number for the backlight. In this case, the light will be on once the `BacklightBrightness` property if more then 0.5, otherwise off.
+
+### Displaying text
+
+Like for other screen bindings in this repository you have `Write` and `SetCursorPosition` to write text and set the cursor position.
+
+A specific font is available and optimized for the screen. The font only contains a limited number of characters and will just ignore any characters that are unknown.
+
+```csharp
+lcd.SetCursorPosition(0, 0);
+lcd.WriteLine("First line");
+lcd.Write("Second one");
+lcd.SetCursorPosition(0, 5);
+lcd.Write("last line");
+```
+
+The position of the cursor moves with the text you write. The screen has 6 lines in total. And a character is 5 pixels width and 8 pixels height.
+
+You can as well provide your own character using the `Write(ReadOnlySpan<byte> text)` function.
+
+### Drawing lines, rectangles and points
+
+Point, line and rectangle primitives are available:
+
+```csharp
+lcd.DrawPoint(5, 5, true);
+lcd.DrawLine(0, 0, 15, 35, true);
+lcd.DrawRectangle(10, 30, 10, 20, true, true);
+lcd.DrawRectangle(12, 32, 6, 16, false, true);
+// You should not forget to draw what you have in memory
+lcd.Draw();
+```
+
+Each can take `Point`, `Size` and `Rectangle` as well as input. You have to decide if you want to have the point on or off and if you fill or not the rectangle.
+
+Also you should `Refresh` the screen once you'll finish your drawing. All drawing are in the memory and needs to be pushed to the screen.
+
+### Displaying raw images
+
+The function SetByteMap allows you to draw anything, you'll just need to provide the raw buffer. Size is 504 bytes representing from the top left part columns of 8 pixels up to the right up to the next raw. A total of 6 raws are available with 84 columns each.
+
+Here is an example on how you can convert an existing image to extract the raw data:
+
+```csharp
+using Image<Rgba32> bitmapNokia = Image.Load<Rgba32>(Path.Combine("nokia_bw.bmp"));
```

Comment:
I guess you have to replace this part of the readme with your application Img2Pcd8544 code